README.md

```diff
@@ -6,6 +6,23 @@ REST APIs using Jersey, Jackson, Jetty, and Hibernate Validator.
 
 See the [`examples/`](examples/) directory for a simple demo service.
 
+Security
+--------
+The REST `Application` can support both http and https. https is disabled by default. Three modes are supported:
+http, https, and http+https. In the last mode, http and https are supported on different ports.
+
+The https implementation is similar to Kafka, where the server specifies a keystore and trust store. When SSL
+client auth is configured, the client must authenticate -- the key in the client's keystore must be trusted by
+the server's trust store, either because the client cert is in the server's trust store, or because the CA that signed
+the client's cert is in the server's trust store.
+
+Metrics
+-------
+When both http and https protocols are enabled, two metrics groups are created, `jetty-metrics` and
+`jetty-https-metrics`. However, `jetty-https-metrics` is a duplicate of `jetty-metrics`. Put differently, each
+metrics group will have the same value for all metrics, and those values will be influenced by both http and
```

Comment:
Hmm, not sure I understand the purpose of doing this? If they are identical, couldn't we just stick with one? I would get the second group if we split the accounting of traffic between them, but this setup seems... wasteful?